Firebase/Auth/Source/Public/FIRAuthDataResult.h

```diff
@@ -37,7 +38,14 @@ NS_SWIFT_NAME(AuthDataResult)
 /** @property user
     @brief The signed in user.
  */
-@property(nonatomic, readonly) FIRUser *user;
+@property(nonatomic, readonly, nullable) FIRUser *user;
+
+/** @property credential
+    @brief The updated OAuth credential after the the sign-in, link and reauthenticate action.
+    @detial This property is for OAuth sign in only.
+ */
+@property(nonatomic, readonly, nullable) FIROAuthCredential *credential;
```

Comment:
This API was approved and both Web and Android has it, but somehow iOS missed it before I work here.
So I added it